internal/handlers/handlers.go

```diff
@@ -1 +1,49 @@
 package handlers
+
+import (
+	"bufio"
+	"fmt"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/s444v/go-sixth-sprint/internal/service"
+)
+
+func MainHandler(w http.ResponseWriter, r *http.Request) {
+	content, err := os.ReadFile("index.html")
+	if err != nil {
+		http.Error(w, "Cant read file", http.StatusInternalServerError)
```

Comment:
Это скорее ошибка клиента
```suggestion
		http.Error(w, "Cant read file", http.StatusBadRequest)
```